pipelines/resource_pipeline.go

```diff
@@ -346,7 +346,7 @@ func adjustPipelineResourceSchema(m map[string]*schema.Schema) map[string]*schem
 	m["storage"].DiffSuppressFunc = suppressStorageDiff
 	m["storage"].ConflictsWith = []string{"catalog"}
 	m["catalog"].ConflictsWith = []string{"storage"}
-
+	m["name"].DiffSuppressFunc = common.EqualFoldDiffSuppress
```

Comment:
Hmm, was it checked with the DLT team?

Reply:
I have asked with the team for confirmation.

Reply:
removed it for the time being